content/en-us/tutorials/core/building/create-basic-visual-effects.md

```diff
@@ -71,7 +71,7 @@ Each particle displays an image set by the `Class.ParticleEmitter.Texture|Textur
 
 You can use Roblox's pre-made flare image for the `Class.ParticleEmitter.Texture|Texture` of the particle emitter. To use Roblox's pre-made asset:
 
-1. In the **Explorer** window, select **Emitter_Flare**.
+1. In the **Explorer** window, select **Emitter_Flare** > **Attachment** > **ParticleEmitter**.
```

Comment:
I see that you tool a different approach here to call the part Emitter_Flare, and kept the particle emitter without a name. This is a valid approach too! To keep the content consistent with the sample place file, we're going to keep the content as-is.